.github/workflows/maven-test.yml

```diff
@@ -14,7 +14,8 @@ on:
 
 jobs:
   maven-tests:
-    uses: wultra/wultra-infrastructure/.github/workflows/maven-test.yml@develop
+    uses: wultra/wultra-infrastructure/.github/workflows/maven-test.yml@issues/persist-maven-test-resource
     secrets: inherit
     with:
-      java_version: 21
\ No newline at end of file
+      java_version: 21
+      artifacts_path: '**/signer-cloud-server/target/test-resources'
```

Comment:
🤔 I am not sure whether it is needed to archive the keystore at GHA. It is great that you can use the reproducible input in the local environment.
If you prefer to keep it at GHA, do we have to update our general `maven-test.yaml`? Isn't it possible to modify it only here and extract after there is more than one usage?

Reply:
It is up to you. IMHO, it is not necessary to store that file, but feel free to add it.